lib/engine/step/special_track.rb

```diff
@@ -165,7 +166,7 @@ def check_connect(_action, ability)
         return if ability.hexes.size < 2
         return if !ability.start_count || ability.start_count < 2 || ability.start_count == ability.count
 
-        paths = ability.hexes.flat_map do |hex_id|
+        paths = ability.laid_hexes.flat_map do |hex_id|
```

Comment:
With a change to one of the base steps, I was a little bit nervous but validated against the full DB and it's all good. 👍